<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_num7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
pressed{background-color: rgb(224, 0, 0)};</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
padding-right:  85px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_comma">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
 background-color: rgba(52,52,52,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(52,52,52,255);
</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_division">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>40</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgba(255,158,9,255);
color: white;
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(255,158,9,255);
text-align: top;
</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_multiplication">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>40</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgba(255,158,9,255);
 border-style: solid;
 border-width:1px;
color: white;

 border-radius:35px;
 border-color: rgba(255,158,9,255);
text-align: top;
</string>
    </property>
    <property name="text">
     <string>ร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_subtraction">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>40</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgba(255,158,9,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(255,158,9,255);
text-align: top;
color: white;
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addition">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>40</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
 background-color: rgba(255,158,9,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(255,158,9,255);
text-align: top;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_result">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>520</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>40</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgba(255,158,9,255);
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 border-color: rgba(255,158,9,255);
text-align: top;
color: white;
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_AC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgba(165,165,165,255);
 border-style: solid;
color: black;
 border-width:1px;
 border-radius:35px;

</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plusoverminus">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgba(165,165,165,255);
 border-style: solid;
color: black;
 border-width:1px;
 border-radius:35px;


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Media/Screenshots/Selection_141.png</normaloff>../../../Media/Screenshots/Selection_141.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgba(165,165,165,255);
 border-style: solid;
color: black;
 border-width:1px;
 border-radius:35px;

</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
